src/shared/services/commandInterpreterHelper.js

```diff
@@ -195,5 +195,6 @@ const interpret = (cmd) => {
 }
 
 export default {
-  interpret
+  interpret,
+  commands: availableCommands.map((command) => command.name)
```

Comment:
Yeap. In general I thing the solution should somehow retrieve actual commands from the application, rather than having them hard-coded.

On top of that it would be handy to have subcomads as well (e.g. For help). We have theoretical possibility to implement autocompletion for nested commands.

We did something like that for procedure call output parameters.